src/test/java/uk/gov/hmcts/juror/api/moj/report/standard/AbaccusReportTest.java

```diff
@@ -0,0 +1,147 @@
+package uk.gov.hmcts.juror.api.moj.report.standard;
+
+import com.querydsl.core.Tuple;
+import com.querydsl.jpa.impl.JPAQuery;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.StandardReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.AbstractReportResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardReportResponse;
+import uk.gov.hmcts.juror.api.moj.domain.QBulkPrintData;
+import uk.gov.hmcts.juror.api.moj.report.AbstractStandardReportTestSupport;
+import uk.gov.hmcts.juror.api.moj.report.DataType;
+import uk.gov.hmcts.juror.api.moj.repository.PoolRequestRepository;
+
+import java.time.Clock;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@SuppressWarnings("PMD.LawOfDemeter")
+public class AbaccusReportTest extends AbstractStandardReportTestSupport<AbaccusReport> {
+
+    private final Clock clock;
+
+
+    public AbaccusReportTest() {
+        super(
+            QBulkPrintData.bulkPrintData,
+            AbaccusReport.RequestValidator.class,
+            DataType.DOCUMENT_CODE,
+            DataType.TOTAL_SENT_FOR_PRINTING,
+            DataType.DATE_SENT);
+        setHasPoolRepository(false);
+        String instantExpected = "2024-04-30T12:00:01Z";
+        Instant instant = Instant.parse(instantExpected);
+
+        this.clock = mock(Clock.class);
+        when(clock.instant()).thenReturn(instant);
+        when(clock.getZone()).thenReturn(ZoneId.systemDefault());
```

Comment:
Not needed as we can just verify the time is within 5 mins

Reply:
Removed